test/exe/Main.hs

```diff
@@ -2241,6 +2244,20 @@ nonLspCommandLine = testGroup "ghcide command line"
         ec @=? ExitSuccess
   ]
 
+benchmarkTests :: TestTree
+benchmarkTests =
+    let ?config = Bench.defConfig
+            { Bench.verbosity = Bench.Quiet
+            , Bench.repetitions = Just 3
+            , Bench.buildTool = Bench.Stack
+            } in
+    withResource Bench.setup id $ \_ -> testGroup "benchmark experiments"
+    [ testCase (Bench.name e) $ do
+        res <- Bench.runBench runInDir e
+        assertBool "did not successfully complete 5 repetitions" $ Bench.success res
```

Comment:
This has just failed on master (no additional output). Do you expect this to be flaky?

Reply:
For me it was failing locally because it ran stack, which then tried to download `ghc-8.6.5`, which led to a timeout.

Reply:
This is flaky, yes. I'll have a fix in a few minutes

Reply:
It defaults to `stack` because that's what CI uses, but it can be made to work with Cabal as well. I just haven't figured out how to autodetect which build tool is being used